### PR TITLE
6070: lambda, proxy, interactor and tests to seal a case

### PR DESCRIPTION
**Pull request by kkoskelin**

### CODE REVIEW

File: shared/src/business/useCases/sealCaseContactAddressInteractor.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+} = require('../../errors/errors');
+const { Case } = require('../entities/cases/Case');
+
+/**
+ * sealCaseContactAddressInteractor
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.applicationContext the application context
+ * @param {string} providers.docketNumber the docket number of the case to update
+ * @param {object} providers.caseToUpdate the updated case data
```
Comment (matthopson) [inline shared/src/business/useCases/sealCaseContactAddressInteractor.js]: jsdoc

File: shared/src/business/useCases/sealCaseContactAddressInteractor.test.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      role: ROLES.docketClerk,
+      userId: 'docketClerk',
+    });
+    await expect(
+      sealCaseContactAddressInteractor({
+        applicationContext,
+        contactId: '23-skidoo',
+        docketNumber: MOCK_CASE.docketNumber,
+      }),
+    ).rejects.toThrow('Cannot seal contact');
```
Comment (matthopson) [inline shared/src/business/useCases/sealCaseContactAddressInteractor.test.js:39]: Looks like it's throwing more than this

**Review by matthopson — APPROVED** [review]
sealwithit

File: shared/src/business/useCases/sealCaseContactAddressInteractor.test.js (hunk 2 above)
Comment (kkoskelin) [reply shared/src/business/useCases/sealCaseContactAddressInteractor.test.js:39]: Yep. this check passes when it simply matches a substring -- which is enough for me to know that the correct condition is responsible for rejecting the promise.

Comment (matthopson) [reply shared/src/business/useCases/sealCaseContactAddressInteractor.test.js:39]: Ah, that's neat. Did not know that. Thanks!

**Pull request merged**